tests/tooltip.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { test, expect } from '@playwright/test';
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('/tooltip-test.html');
+});
+
+test('Confirms tooltip text is visible upon hover', async ({ page }) => {
+  const tooltip = page.getByRole('tooltip', { name: 'This is the hover test'})
+  await expect(tooltip).not.toBeVisible()
+  const wordToHover = page.getByText('Hover Test').first()
+  await wordToHover.hover();
+  await expect(tooltip).toBeVisible()
+});
+
+test('Confirms tooltip text is above text to hover if location attribute is specified as "top"', async ({ page }) => {
+  const text = page.locator('p:near(:text("This is the top test"))')
+  const tooltip = page.getByRole('tooltip', { name: 'This is the top test'})
+  const wordToHover = page.getByText('Top Test').first()
+
+  const textLocation = await text.evaluate(element => element.getBoundingClientRect());
+  wordToHover.hover()
+  const tooltipLocation = await tooltip.evaluate(element => element.getBoundingClientRect());
+  
+  expect(tooltipLocation.y).toBeLessThan(textLocation.y)
```

Comment:
Just a heads up that some properties provided by getBoundingClientRect (x, y, top, bottom, left, right values) are relative to the top-left of the viewport.  Normally, you would think that tooltip location y would be greater than the hover text location y when you're expecting the tooltip to be above the hover text.  But in this case, it's the opposite.